test/VaultOUSD.ts

```diff
@@ -1,12 +1,12 @@
 import { expect } from "chai";
```

Comment:
All changes in this file are to remove `deploy` with missing argument (admin address) and update references to point to ContractTestContext.